Test.Integration.Net7/EventsAPI/Promotions.cs

```diff
@@ -0,0 +1,65 @@
+using Sift;
+using System.Collections.ObjectModel;
+using Test.Integration.Net7.Uitlities;
+using Xunit;
+
+namespace Test.Integration.Net7.EventsAPI
+{
+    public class Promotions
```

Comment:
If a class contains just one test case it is ok to name it similarly to the event type. In this case it can be `AddPromotionTest`. The same we can do for `Notifications` -> `SecurityNotificationTest`, `Chargebacks` -> `ChargebackTest` etc